Dockerfile

```diff
@@ -7,12 +7,14 @@ RUN curl --location https://www.phpmyadmin.net/downloads/phpMyAdmin-latest-all-l
  && rm -rf /www/js/jquery/src/ /www/examples /www/po/
 
 COPY config.inc.php /www/
-COPY run.sh /run.sh
-RUN chmod u+rwx /run.sh
+COPY docker-entrypoint.sh /docker-entrypoint.sh
```

Comment:
chmod is not needed anymore? It seems that at least in some setups it is, see eg. https://github.com/klaemo/docker-couchdb/issues/19